.github/workflows/deployment.yaml

```diff
@@ -0,0 +1,30 @@
+name: Deployment
+
+on:
```

Comment:
This will be run at the same time as the development QA when pushed to master. Add !main to ignore the main branch.